### ISSUE TITLE
Invalid Parsing of pepXML from msFragger Output

### ISSUE THREAD

**Issue by smithi2** [chhh/MSFragger-GUI#13]
Hi there,

I have been working on using the non-GUI version of msFragger on a phospho-enriched sample. The search completes and I end up getting a pepXML output, however when I attempt to open the file via TPP or other parsing tools I end up getting a parsing error. For TPP, I receive the following error:
unable to read pepxml file: error parsing pepxml file: XML parsing error: not well-formed (invalid token), at xml file line 20, column 333.

Best,
Ian

**Comment by chhh** [comment]
So what was the problem? I guess you've figured it out since you closed the ticket.

**Comment by smithi2** [comment]
Yes, I was using an SGD yeast database that was not being parsed properly.
I then used a UniProt based database and software worked well and I had not
problem bringing the pepXML output into TPP. All good to go now. Thank you.
